index.html

```diff
@@ -1,80 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rock, Paper, Scissors</title>
-</head>
+  </head>
 
-<body>
+  <body>
 
-<script>
-  
-function computerPlay() {
-    let rand = Math.random();
-    if (rand < 0.34) {
+    <script>
+    
+    function computerPlay() {
```

Comment:
<script> is the parent for your JS code, so all the js code will need to be indented one more